comments.go

```diff
@@ -72,6 +72,34 @@ func (c *CommentsController) Update(ctx *app.UpdateCommentsContext) error {
 	})
 }
 
+// Delete does DELETE comment
+func (c *CommentsController) Delete(ctx *app.DeleteCommentsContext) error {
+	id, err := uuid.FromString(ctx.ID)
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound(err.Error()))
+	}
+	identity, err := login.ContextIdentity(ctx)
+	if err != nil {
+		return jsonapi.JSONErrorResponse(ctx, goa.ErrUnauthorized(err.Error()))
+	}
+
+	return application.Transactional(c.db, func(appl application.Application) error {
+		cm, err := appl.Comments().Load(ctx.Context, id)
+		if err != nil {
+			return jsonapi.JSONErrorResponse(ctx, err)
+		}
+		if identity != cm.CreatedBy.String() {
```

Comment:
`master` branch now also contains a `comments_blackbox_test.go` file which you could use to verify that a user cannot delete a comment that she/he did not author.